modules/server_license.py

```diff
@@ -0,0 +1,20 @@
+from prometheus_client import Metric
+
+
+class UcsServerLicense:
```

Comment:
This is basically the UcsServerLicenseCollector collector already. Please also treat it accordingly.

Reply:
Not sure exactly, what you want to say. I renamed it as collector